pkg/injector/monitoring/metrics.go

```diff
@@ -0,0 +1,85 @@
+// ------------------------------------------------------------
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+// ------------------------------------------------------------
+
+package monitoring
+
+import (
+	"context"
+
+	"go.opencensus.io/stats"
+	"go.opencensus.io/stats/view"
+	"go.opencensus.io/tag"
+)
+
+const (
+	appID   = "app_id"
+	podName = "pod_name"
+)
+
+var (
+	successfulAdmissionReviewResponseTotal = stats.Int64(
+		"injector/successful_admissionreview_response_total",
+		"The total number of successful admission review responses.",
+		stats.UnitDimensionless)
+	failedAdmissionReviewResponseTotal = stats.Int64(
+		"injector/failed_admissionreview_response_total",
+		"The total number of failed admission review responses.",
+		stats.UnitDimensionless)
+
+	// appIDKey is a tag key for App ID
+	appIDKey = tag.MustNewKey(appID)
```

Comment:
appIDKey will be duplicated info if we track podName because podName represent its app name as well.